mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -246,6 +248,7 @@ private HashMap<String, URIResolver> getResolverMapping() {
         supportedSchemes.put("https", restResolver);
         supportedSchemes.put("file", new MCRFileResolver());
         supportedSchemes.put("cache", new MCRCachingResolver());
+        supportedSchemes.put("function", new MCRFunctionResolver());
```

Comment:
I would like to have a more precise name than simply `function:`
What about `java:` or `java-call:`

Reply:
In xed forms one can use `call-java`. Maybe we should stick with that.

Reply:
sound good for me

Reply:
@rsteph-de update schema to `call-java`

Reply:
Cool!